server/sonar-web/src/main/js/apps/issues/sidebar/ProjectFacet.js

```diff
@@ -28,7 +28,8 @@ import FacetFooter from './components/FacetFooter';
 import type { ReferencedComponent, Component } from '../utils';
 import Organization from '../../../components/shared/Organization';
 import QualifierIcon from '../../../components/shared/QualifierIcon';
-import { searchComponents, getTree } from '../../../api/components';
+import { convertToFilter } from '../../../apps/projects/store/utils';
```

Comment:
I'd not introduce a dependency between `issues` and `projects` apps for such small detail